### PR TITLE
chore: update pp and remove redundant tests

### PR DESCRIPTION
**Pull request by Alizter**
Signed-off-by: Ali Caglayan <alizter@gmail.com>

### CODE REVIEW

File: vendor/update-pp.sh [hunk 1]
```diff
@@ -20,6 +21,8 @@ mkdir -p pp/src
 SRC=$TMP/pp
 
 cp -v $SRC/src/pp.{ml,mli} pp/src
+cp -v $SRC/test/tests.ml pp/test/pp_tests.ml
```
Comment (rgrinberg) [inline vendor/update-pp.sh:24]: why do we rename the tests? seems like a gratuitous change.

Comment (Alizter) [reply vendor/update-pp.sh:24]: I wanted to preserve the name from before, but I can get rid of it.

Comment (rgrinberg) [reply vendor/update-pp.sh:24]: I say we should stay as close to upstream as possible

File: vendor/update-pp.sh [hunk 2]
```diff
@@ -20,6 +21,8 @@ mkdir -p pp/src
 SRC=$TMP/pp
 
 cp -v $SRC/src/pp.{ml,mli} pp/src
+cp -v $SRC/test/tests.ml pp/test/tests.ml
```
Comment (rgrinberg) [inline vendor/update-pp.sh:24]: Where is tests.ml?

Btw, there's actually no need to vendor the test at all. It's enough to run the tests in ocaml-dune/pp

Comment (Alizter) [reply vendor/update-pp.sh:24]: We don't run it there AFAICT.

Comment (Alizter) [reply vendor/update-pp.sh:24]: I didn't rerun the script, let me try again.

Comment (rgrinberg) [reply vendor/update-pp.sh:24]: We won't run it here either. We don't run tests in the vendored directory. So we might as well stop copying it.

Comment (Alizter) [reply vendor/update-pp.sh:24]: OK I will make a PR adding some basic github actions to the fork.

Comment (Alizter) [reply vendor/update-pp.sh:24]: and I'll drop the tests

**Comment by Alizter** [comment]
I've added a workflow to the pp repo: https://github.com/ocaml-dune/pp/pull/7
